static/docs/use-cases/how-to-store-files-in-dvc.md

```diff
@@ -0,0 +1,129 @@
+# How to store files in DVC
+
+DVC allows storing and versioning files with git, without checking the file
```

Comment:
Let's use capital letter "Git" instead of "git".

Reply:
Ok.